dlrover/python/diagnosis/common/constants.py

```diff
@@ -48,3 +48,10 @@ class DiagnosisActionType(object):
     # node operation
     RESTART_WORKER = "restart_worker"
     RELAUNCH_WORKER = "relaunch_worker"
+
+    ACTION_TYPE_ANY = "any"
+    ACTION_TYPE_LOG = "log"
+
+    ACTION_EXPIRED_TIME_PERIOD = 60 * 5
```

Comment:
'ACTION_EXPIRED_TIME_PERIOD' is already defined in constant, and it's not a action type.